local types = require(script.types)

export type Quaternion = types.Quaternion

return table.freeze({
	angle = require(script.angle),
	applyRotation = require(script.applyRotation),
	dot = require(script.dot),
	equals = require(script.equals),
	fromAxisAngles = require(script.fromAxisAngles),
	fromCFrame = require(script.fromCFrame),
	fromEulerAngles = require(script.fromEulerAngles),
	fromToRotation = require(script.fromToRotation),
	fuzzyEquals = require(script.fuzzyEquals),
	getComponents = require(script.getComponents),
	identity = require(script.identity),
	inverse = require(script.inverse),
	inverseInPlace = require(script.inverseInPlace),
	inverseUnit = require(script.inverseUnit),
	inverseUnitInPlace = require(script.inverseUnitInPlace),
	lookAlong = require(script.lookAlong),
	lookAt = require(script.lookAt),
	lookVector = require(script.lookVector),
	magnitude = require(script.magnitude),
	magnitudeSquared = require(script.magnitudeSquared),
	multiply = require(script.multiply),
	multiplyInPlace = require(script.multiplyInPlace),
	new = require(script.new),
	nlerp = require(script.nlerp),
	nlerpInPlace = require(script.nlerpInPlace),
	normalize = require(script.normalize),
	normalizeInPlace = require(script.normalizeInPlace),
	rightVector = require(script.rightVector),
	rotateTowards = require(script.rotateTowards),
	rotateTowardsInPlace = require(script.rotateTowardsInPlace),
	rotateVector = require(script.rotateVector),
	rotation = require(script.rotation),
	set = require(script.set),
	setAxisAngle = require(script.setAxisAngle),
	setCFrame = require(script.setCFrame),
	setEulerAngles = require(script.setEulerAngles),
	setFromToRotation = require(script.setFromToRotation),
	setLookRotation = require(script.setLookRotation),
	slerp = require(script.slerp),
	slerpFromIdentity = require(script.slerpFromIdentity),
	slerpFromIdentityInPlace = require(script.slerpFromIdentityInPlace),
	slerpInPlace = require(script.slerpInPlace),
	toAxisAngle = require(script.toAxisAngle),
	toCFrame = require(script.toCFrame),
	toCFrameFromPosition = require(script.toCFrameFromPosition),
	toEulerAngles = require(script.toEulerAngles),
	tostring = require(script.tostring),
	upVector = require(script.upVector),
	w = require(script.w),
	x = require(script.x),
	xVector = require(script.xVector),
	y = require(script.y),
	yVector = require(script.yVector),
	z = require(script.z),
	zVector = require(script.zVector),
})
